tests/snuba/sessions/test_sessions.py

```diff
@@ -1,1590 +0,0 @@
-from __future__ import annotations
```

Comment:
This one should not be removed but rather we should keep the tests for metrics only. I will take care of this.